packages/manager/modules/pci/src/projects/project/public-gateways/translations/Messages_fr_FR.json

```
@@ -1,3 +1,14 @@
 {
-  "pci_projects_project_public_gateway_title": "Public Gateways"
+  "pci_projects_project_public_gateway_title": "Public Gateways",
+  "pci_projects_project_public_gateways_intro": "Les Public Gateway fournissent une connectivité aux instances privées, permettant d’accéder de manière sécurisée aux réseaux externes ou d’exposer l'application à un accès externe:<ul><li>Dans le premier cas, Public Gateway agit comme une passerelle SNAT, autorisant les instances à atteindre Internet, mais pas le contraire.</li><li>Dans le second exemple, il s’agit de donner la possibilité d’exposer un service derrière une instance privée ou un Load Balancer à Internet en utilisant des IP flottantes.</li></ul>",
+  "pci_projects_project_public_gateway_create": "Créer une Public Gateway",
+  "pci_projects_project_public_gateway_name": "Nom",
+  "pci_projects_project_public_gateway_region": "Région",
+  "pci_projects_project_public_gateway_networks_connected": "Réseau connecté",
+  "pci_projects_project_public_gateway_public_ip": "IP publique",
+  "pci_projects_project_public_gateway_flavour": "Taille",
+  "pci_projects_project_public_gateway_status": "Statut",
+  "pci_projects_project_network_private_modify": "Modify Flavour",
```

```suggestion
  "pci_projects_project_network_private_modify": "Editer l'instance",
```
? Might need to check what is the wanted translation for this